[cert-manager.md] Improve `ClusterIssuer` snippet (#3707)
Kong/docs.konghq.com: app/kubernetes-ingress-controller/2.1.x/guides/cert-manager.md

    solvers:
    - http01:
        ingress:
          podTemplate:
             metadata:
               annotations:

>                  kuma.io/sidecar-injection: "false"   # If ingress is running in Kuma/Kong Mesh, disable sidecar injection

Reviewer: ```suggestion
                 kuma.io/sidecar-injection: "false"   # If ingress is running in Kuma/Kong Mesh, disable sidecar injection
                 sidecar.istio.io/inject: "false"
```

Reviewer 2: Yup!

>                  kuma.io/sidecar-injection: "false"   #in case of ingress will be running in Kuma/Kong Mesh, disable sidecar injection

Reviewer: ```suggestion
                 kuma.io/sidecar-injection: "false"   # If ingress is running in Kuma/Kong Mesh, disable sidecar injection
```
